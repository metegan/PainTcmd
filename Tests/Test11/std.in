C c1 2 2 2
C c2 3 3 3
OA o1 c1 c2
OA o2 o1 c1
OA o3 o1 o2
DELETE c2
LIST
UNDO
LIST
EXIT

