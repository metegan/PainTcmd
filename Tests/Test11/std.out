OK
# New object : c1
OK
# New object : c2
OK
# New object : o1
OK
# New object : o2
OK
# New object : o3
OK
# Object c2 deleted
C c1 2 2 2
OK
OA o1 c1 c2
OA o3 o2 o1
OA o2 c1 o1
C c2 3 3 3
C c1 2 2 2

